themes/custom/ts_wrin/sass/components/_region-map.scss

```diff
@@ -46,13 +52,27 @@
   }
 }
 
+.interactive-map {
+  width: 100%;
+  height: auto;
+  background-color: $white;
```

Comment:
I should have warned you that [`/our-work](https://pr-1256-wriflagship.pantheonsite.io/our-work)` also has the map, but its background is grey there. There are two variations right now based on the parent class

`has-light-gray-background-color` and `has-light-gray-background-color`

Can you update that?

Reply:
Done!